### PR TITLE
Info boxes

### PR DESCRIPTION
**Pull request by Scott-Crawshaw**
Info boxes branch should be ready for a merge. Info boxes & their corresponding info are put into place, and a front page carousel with more details on the project is in place as well. Further steps might include coming up with more info box placements, and creating a tutorial walkthrough, so let's leave the branch open. I don't think this will break anything, but double check to make sure. I'll finalize the merge in class today

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export const API_URL = 'https://med30.herokuapp.com/';
-// export const API_URL = 'http://localhost:3838/';
+// export const API_URL = 'https://med30.herokuapp.com/';
+export const API_URL = 'http://localhost:3838/';
 export const CALL = 'CALL';
```
Comment (hack-heart) [inline src/constants.js:3]: Q: why this change?

File: src/common/InfoText.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
+// Front Page:
+export const frontPage = 'This application aims to discuss the inefficiencies in the storage and retrieval of medical record data. \
```
Comment (hack-heart) [inline src/common/InfoText.js:2]: FIX: These strings shouldn't be variables. Most things on a website are strings, so it's much easier to place the strings where they should be. The only alternative is if there are multiple questions/answers all in the same format, then placing them in a JS object/array or a JSON file would make mapping them to a React component or fragment easier.

File: src/home/LandingPage.js [hunk 3]
```diff
@@ -3,8 +3,12 @@ import { Link } from 'react-router-dom';
 import Button from '@mui/material/Button';
 import Typography from '@mui/material/Typography';
 import Box from '@mui/material/Box';
+import Carousel from 'react-bootstrap/Carousel';
```
Comment (hack-heart) [inline src/home/LandingPage.js:6]: NIT: (Nits are generally my personal opinion/suggestions). Most users don't bother scrolling through carousels. Prefer to display info vertically, especially since the landing page is bare.

File: src/home/LandingPage.js [hunk 4]
```diff
@@ -3,8 +3,12 @@ import { Link } from 'react-router-dom';
...
 import Header from '../common/Header';
 import image from '../images/landing_img.jpg';
+import {
+  probPrompt, prob1, prob1Content, prob2, prob2Content, prob3, prob3Content, prob4, prob4Content, sol1Content, sol2Content, sol3Content, sol4Content,
```
Comment (hack-heart) [inline src/home/LandingPage.js:10]: This is how tedious importing can get when there are many variables.

File: src/index.js [hunk 5]
```diff
@@ -7,6 +7,7 @@ import App from './app';
 import store from './app/store';
 import theme from './theme';
 import reportWebVitals from './reportWebVitals';
+import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (hack-heart) [inline src/index.js:10]: NIT: Project already uses MUI, Bootstrap is a different look and feel.

File: src/patient/PatientDoctors.js [hunk 6]
```diff
@@ -114,6 +116,7 @@ const PatientDoctors = () => {
           </TableBody>
         </Table>
       </TableContainer>
+      <InfoPopover style={{ marginTop: '15px', marginBottom: '5px' }}>{addDoctorInfo}</InfoPopover>
       <Button onClick={handleOpen} disableElevation disableRipple disableFocusRipple style={btnStyle} sx={{ flexDirection: 'column' }}>
```
Comment (hack-heart) [inline src/patient/PatientDoctors.js:120]: Q: Is this part of the tutorial?
FIX: Place above table

**Review by hack-heart — CHANGES_REQUESTED** [review]

File: src/patient/PatientDoctors.js (hunk 6 above)
Comment (Scott-Crawshaw) [reply src/patient/PatientDoctors.js:120]: Moved it, thanks! Is what part of the tutorial? There wasn't much of a tutorial, so likely no

File: src/constants.js (hunk 1 above)
Comment (Scott-Crawshaw) [reply src/constants.js:3]: Good call, I'll change that back

File: src/common/InfoText.js (hunk 2 above)
Comment (Scott-Crawshaw) [reply src/common/InfoText.js:2]: Sounds good, I'll move the strings into the code. I will leave this file though, as some of the strings have yet to be used, so we want to keep the content for later.

Comment (Scott-Crawshaw) [reply src/common/InfoText.js:2]: Actually we decided to keep it for now, we worked like this because Kevin was making/adjusting the text while I was coding the boxes, and since we will continue doing that, it's easier for now. Am happy to move these though down the road, just wanna get this merged in during our in class meeting

**Comment by Scott-Crawshaw** [comment]
@hack-heart Thanks for the thorough comments, much appreciated! I moved the info box on the doctors page and reverted the constants file. Kevin and I have a workflow going with the variables file (he writes and edits the text, I place the boxes) so it's easiest to keep that file for now, but am happy to move the strings into the code down the road. Let me know if there are any issues
